### PR TITLE
Add pages, react router, and styles

### PR DESCRIPTION
**Pull request by dennis-every**
In this pull request, we completed the following per the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website_v2.md):

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- Style the general layout according to the wireframes.
- Add styles to improve the look and feel.
- Use React basic styling (inline styles, importing CSS, and pre-processed CSS files).
- For every page, you should create a Route.
- Make sure that navigation links for all 3 routes are displayed on each page.
- Use React Router to make the routes work.

### CODE REVIEW

File: src/routes/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* eslint-disable max-len */
+import './Home.css';
+
```
Comment (TadesseDev) [inline src/routes/Home.jsx]: - [ ] Dear @dennis-every, Your project has been set up well, but It's not recommended to disable `linters` in our code base, Linters are here to make sure our code is clean and according to the standards. Pleae, remove disabled linter rules.

File: src/routes/Quote.jsx [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/* eslint-disable max-len */
+import './Quote.css';
```
Comment (TadesseDev) [inline src/routes/Quote.jsx]: - [ ] Dear @dennis-every, It's not recommended to disable `linters` in our code base, Linters are here to make sure our code is clean and according to the standards. Pleae, remove disabled linter rules.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @dennis-every 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7


## Highlights
 - [ ] Good job overall following the template design ✅
 - [ ]  Correct GitFlow  ✅
 - [ ]  Good commit history  ✅
 - [ ]  Good PR title/description  ✅
 - [ ]  All linters passed  ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/routes/Quote.jsx (hunk 2 above)
Comment (dennis-every) [reply src/routes/Quote.jsx]: Hi @TadesseDev, ok, tks for the feedback!

**Review by m05e5 — APPROVED** [review]
**Hi @dennis-every 
You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights

✔️ No linters error
✔️  Well done implementing the requested changes 
✔️ Nice UI

## Optional suggestions



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**

**Pull request merged**